fake-theta/commands/listFiles.js

```diff
@@ -1 +1,15 @@
-TODO: Erik
\ No newline at end of file
+const body = {'name': 'camera.listFiles',
+              "parameters": {
+                  "fileType": "all",
+                  "entryCount": 5,
+                  "maxThumbSize": 0   
+    }
+}
+const response = await fetch('https://fake-theta.vercel.app/osc/commands/execute', 
+	{method: 'POST', 
+	body: JSON.stringify(body),
+	headers: {'Content-Type': 'application/json'}
+});
+const data =  await response.json();
+
+console.log(data);
```

Comment:
use stringify to print out the contents of a JavaScript object to a json string

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify

```javascript
console.log(JSON.stringify(data, null, 4))
```

Refer to the last line of this example:

https://github.com/theta360developers/theta-javascript/blob/main/fake-theta/examples/lastThumbUrl.js